EX02/app/src/main/java/com/example/Game_of_throne/RecyclerViewAdapter1.java

```diff
@@ -0,0 +1,57 @@
+package com.example.Game_of_throne;
```

Comment:
A redundant file? Why do you have both RecyclerViewAdapter1 and RecyclerViewAdapter?